Source/ConversationV1/Models/MessageResponse.swift

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+/** A conversation response. */
+public struct MessageResponse: JSONDecodable {
+    
+    // Context, or state, from the Conversation system
+    public var context:  [String: JSON]?
+    
+    // Output, or response, from the system
+    public var output:   [String: JSON]?
+    
+    // List of intents
```

Comment:
It would be nice if this comment was more descriptive. Unfortunately, I'm not sure it's clear what intents are or how they will be used in the future. In general, though, this comment doesn't really express any information that isn't already specified by its type `[Intent]`.